integration-tests/build.gradle

```diff
@@ -161,7 +161,6 @@ task runAllIntegrationTestsForZoweTesting(type: Test) {
     group "Integration tests"
     description "Run all integration tests for Zowe testing"
 
-    dependsOn startUpCheck
```

Comment:
this is because when zowe tests are run, a different verification mechanism is used?

Reply:
Yes, they are run at the moment by a person and there were issues with the startUpCheck there. 
It verifies the availability of the Discoverable Client which isn't in the environment.